osf/models/node.py

```diff
@@ -778,15 +779,17 @@ def can_edit(self, auth=None, user=None):
 
     def get_aggregate_logs_query(self, auth):
         return (
-            (   # TODO: remove `list` call during phase 2
-                Q('node_id', 'in', list(Node.objects.get_children(self).can_view(user=auth.user, private_link=auth.private_link).values_list('id', flat=True))) |
-                Q('node_id', 'eq', self.id)
-            ) & Q('should_hide', 'eq', False)
+            (
+                Q(node_id__in=list(Node.objects.get_children(self).can_view(user=auth.user, private_link=auth.private_link).values_list('id', flat=True))) |
```

Comment:
hmmm.... removing the list call here gives the error:

```
api_1               | ProgrammingError: invalid reference to FROM-clause entry for table "osf_abstractnode"
api_1               | LINE 2:                 "osf_abstractnode".id in (
api_1               |                         ^
api_1               | HINT:  Perhaps you meant to reference the table alias "u0".
```

I worked on this with @sloria for quite awhile and should have done a better job documenting exactly the things I tried and why they didn't work